### DISCUSSION TITLE
posts/2023/08/cross-browser-bookmarks-sync-bookmacster/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#30] [Announcements]
# posts/2023/08/cross-browser-bookmarks-sync-bookmacster/

Needing a reliable way to coordinate all my browsers’ bookmarks, I trial what I learn is a venerable option.

https://www.brycewray.com/posts/2023/08/cross-browser-bookmarks-sync-bookmacster/

**Comment by davidsneighbour** [comment]
Leave the browser and use an external service/program. For instance [Toby](https://www.gettoby.com/) - not sure how it works out on a Mac, but on Ubuntu I have all my bookmarks available, independent of the browser protocol. I am also using [Raindrop](https://raindrop.io) for collecting bookmarks... It's "kind of del.icio.us, but in 2023".

**Reply by brycewray** [reply]
Yeah, the external approach is somewhat similar to how the URL Manager Pro app works, but I didn’t care so much for doing it that way. Also had already decided that, over and above that, Raindrop was simply more than I wanted — **although**, just to be sure before the BookMacster free trial ends (because your comments always make me reconsider stuff after I thought I’d figured them out 😀), I am giving Raindrop’s free tier a try for just the bookmarks.

Also, it appears Toby currently is for only Chrome, Edge, and Firefox, so that would exclude Safari.
